tokio/src/runtime/mod.rs

```diff
@@ -312,6 +312,16 @@ cfg_rt! {
         blocking_pool: BlockingPool,
     }
 
+    /// The flavor of the `Runtime`; the return type for [`Handle::runtime_flavor`](crate::runtime::Handle::runtime_flavor()).
```

Comment:
Stylistic Choice: The `The flavor of the Runtime` text should be on its own line, since that's the line that appears when you preview this item in docs.rs. The part after the semicolon should be on the line after the initial line, in order to prevent the preview from being too long.

Reply:
Yeah, I would also prefer this:
```suggestion
    /// The flavor of a `Runtime`.
    /// 
    /// This is the return type for [`Handle::runtime_flavor`](crate::runtime::Handle::runtime_flavor()).
```

Reply:
Fixed.